### PR TITLE
Bug fix app.openshift.io/vcs-uri and vcs-ref annotations set during i…

### PR DESCRIPTION
**Pull request by vikram-raj**
…mport

Bug fix - https://jira.coreos.com/browse/ODC-1485

Add ```app.openshift.io/vcs-uri``` and ```app.openshift.io/vsc-ref``` annotations for import application.

### CODE REVIEW

File: frontend/packages/dev-console/src/utils/resource-label-utils.ts [hunk 1]
```diff
@@ -8,6 +8,14 @@ export const getAppLabels = (name: string, application: string, imageStreamName?
   };
 };
 
+export const getAppAnnotations = (gitURL: string, gitRef: string) => {
```
Comment (debsmita1) [inline frontend/packages/dev-console/src/utils/resource-label-utils.ts]: please change `gitURL` to `gitUrl`

File: frontend/packages/dev-console/src/utils/resource-label-utils.ts [hunk 2]
```diff
@@ -8,6 +8,14 @@ export const getAppLabels = (name: string, application: string, imageStreamName?
...
+  const ref = gitRef === '' ? 'master' : gitRef;
+  return {
+    'app.openshift.io/vcs-uri': gitURL,
```
Comment (debsmita1) [inline frontend/packages/dev-console/src/utils/resource-label-utils.ts]: please change `gitURL` to `gitUrl`

File: frontend/packages/dev-console/src/utils/resource-label-utils.ts [hunk 3]
```diff
@@ -8,6 +8,14 @@ export const getAppLabels = (name: string, application: string, imageStreamName?
   };
 };
 
+export const getAppAnnotations = (gitURL: string, gitRef: string) => {
+  const ref = gitRef === '' ? 'master' : gitRef;
```
Comment (debsmita1) [inline frontend/packages/dev-console/src/utils/resource-label-utils.ts]:
```suggestion
  const ref = gitRef ? gitRef : 'master';
```

File: frontend/packages/dev-console/src/utils/resource-label-utils.ts (hunk 1 above)
Comment (christianvogt) [reply frontend/packages/dev-console/src/utils/resource-label-utils.ts]: Actually console follows the convention of uppercase acronyms so `gitURL` is correct`.

File: frontend/packages/dev-console/src/utils/resource-label-utils.ts (hunk 3 above)
Comment (christianvogt) [reply frontend/packages/dev-console/src/utils/resource-label-utils.ts]: Or
```suggestion
  const ref = gitRef || 'master';
```

**Comment by christianvogt** [comment]
@vikram-raj what about knative resources ?

**Comment by vikram-raj** [comment]
> @vikram-raj what about knative resources ?

I missed the knative resource I will add it.

**Comment by vikram-raj** [comment]
@christianvogt @debsmita1 Ready for review again.

**Pull request merged**